src/components/common/applicant-info-3.vue

```diff
@@ -126,7 +126,7 @@
 
       <v-row>
         <v-col cols="2" />
-        <v-col cols="5" v-if="showCorpNum">
+        <v-col cols="5" v-if="getShowCorpNum === 'colin'">
```

Comment:
Would you mind adding a const or enum for this value?

See also getter.ts lines 680+ and other code references to this.

(Thanks!)

Reply:
Surreee

Reply:
Updated in next commit! 😁

Reply:
Awesome! Another small step for mankind.